server/script.bat

```diff
@@ -1,13 +1,18 @@
 @echo off
 
-REM GET DIRECTORY
-for %%I in ("%~dp0") do set "SCRIPT_DIR=%%~dpI"
 
-REM GET PARENT DIRS
-for %%I in (%SCRIPT_DIR%) do set "PARENT_DIR=%%~dpI"
+set REPO_URL=https://github.com/SrNightmare09/chess-fen-reader.git
 
-set REPO=chess-fen-reader
+set CLONE_DIR=%CD%
 
-git clone
+set DIR_NAME="%CD%\chess-fen-reader"
 
-start "" "%PARENT_DIR%"
\ No newline at end of file
+git clone %REPO_URL%
+    
+cd %CLONE_DIR%\chess-fen-reader\src\
+
+echo Current directory: %CD%
```

Comment:
I don't believe there is a requirement to print the current directory in the terminal. If there is, please feel free to reply to this change request.